staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -196,6 +196,17 @@ var (
 		[]string{"verb", "resource", "subresource"},
 	)
 
+	requestFilterDuration = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Name: "apiserver_request_filter_duration_seconds",
+			Help: "Request filter latency distribution in seconds, for each filter type",
+			Buckets: []float64{0.0, 0.0001, 0.0002, 0.0004, 0.0008, 0.001, 0.002, 0.004, 0.008, 0.01, 0.015, 0.02, 0.025,
```

Comment:
overall LGTM, except for one nit: 30+ buckets looks too many to me.. isn't 10-ish buckets here just fine?

Reply:
agree - that seems like an overkill to me.
All the filters should actually be pretty fast, so having such a bit granularity for larger values seems a bit useless. Especially for values above a second.

Reply:
I set it to `[]float64{0.0, 0.0005, 0.001, 0.005, 0.01, 0.05, 0.1, 0.25, 0.50, 1.0, 5.0}`. Let me know if this looks good.